clamav-docker-1.4.yaml

```diff
@@ -0,0 +1,97 @@
+package:
+  name: clamav-docker-1.4
+  version: "1.4.1_git20250131"
+  epoch: 0
+  description: An anti-virus toolkit for UNIX eis-ng backport
+  copyright:
+    - license: GPL-2.0-only
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+
+# README
+# these files were grabbed from: https://github.com/Cisco-Talos/clamav-docker/tree/main/clamav/1.0/alpine/scripts
+# This fetch method was chosen because clamav-docker separates their shells scripts out in specific directories
+# upstream also doesn't create tags to track any of these changes, they are tracked mostly in mainline but
+# occasionally tracked on their own branches.
+# If version 1.4 is removed or the has is changed then this build will fail requring manual intervention.
+pipeline:
+  # this is only used for tracking the clamav-docker repo
+  - uses: git-checkout
+    with:
+      repository: https://github.com/Cisco-Talos/clamav-docker
+      expected-commit: 658ba84dfb16f0a70ea6588c5da208e36962e678
+      tag: ""
+      branch: main
+  - uses: fetch
+    with:
+      expected-sha256: 843ec84c968506a7447b4ba137fa0875011f3892b67bebea3ed72abdf84a554e
+      uri: https://raw.githubusercontent.com/Cisco-Talos/clamav-docker/refs/heads/main/clamav/1.4/alpine/scripts/clamdcheck.sh
+      extract: false
+  - uses: fetch
+    with:
+      expected-sha256: f99ed46fb8cd72f899709b652276944ebc5c8d0132890db35154e6dae365814e
+      uri: https://raw.githubusercontent.com/Cisco-Talos/clamav-docker/refs/heads/main/clamav/1.4/alpine/scripts/docker-entrypoint-unprivileged.sh
+      extract: false
+  - uses: fetch
+    with:
+      expected-sha256: 4034f6d63ee6c1d1ed3686733b5722f4b19055b623c82843927613f4e2f7c641
+      uri: https://raw.githubusercontent.com/Cisco-Talos/clamav-docker/refs/heads/main/clamav/1.4/alpine/scripts/docker-entrypoint.sh
+      extract: false
+  - runs: |
+      mkdir -p "${{targets.destdir}}"/usr/bin
```

Comment:
Let's also create `"${{targets.destdir}}"/var/lib/clamav` and `"${{targets.destdir}}"/var/log/clamav` as [they did already](https://github.com/Cisco-Talos/clamav-docker/blob/658ba84dfb16f0a70ea6588c5da208e36962e678/clamav/1.4/debian/Dockerfile#L123C48-L123C63)

Reply:
And also `"${{targets.destdir}}"/run/clamav` and `"${{targets.destdir}}"/tmp` maybe? https://github.com/Cisco-Talos/clamav-docker/blob/658ba84dfb16f0a70ea6588c5da208e36962e678/clamav/1.4/alpine/scripts/docker-entrypoint.sh#L79-L85